dali/test/python/test_backend_impl.py

```diff
@@ -55,9 +55,9 @@ def test_tensorlist_getitem_cpu():
     assert(type(tensorlist[0]) == TensorCPU)
     assert(type(tensorlist[-3]) == TensorCPU)
     assert(len(list_of_tensors) == len(tensorlist))
-    with assert_raises(IndexError):
+    with assert_raises(IndexError, regex="out of range"):
```

Comment:
Why not use glob?

Reply:
Here I have to specify one or the other anyway (unlike assert, it requires a keyword argument), so it's a matter of taste.